{
  assertThrows(log,new Callable<Object>(){
    @Override public Object call() throws Exception {
      data.putAll(null);
      return null;
    }
  }
,NullPointerException.class,"Ouch! Argument cannot be null: entries");
}
